### ISSUE TITLE
DistilBert error

### ISSUE THREAD

**Issue by The-One-Who-Speaks-and-Depicts** [thunlp/VERNet#2]
Currently, when I am attempting at using  DistilBert instead of Bert, there is an error:
```
RuntimeError: The size of tensor a (12) must match the size of tensor b (243) at non-singleton dimension 1
```
How may I fix it?

I am in need of DistilBERT specifically, my hardware is currently unable to cope with BERT.

Note: the packages are indeed of higher versions than recommended, but the code itself runs correctly for BERT (it just dies of CUDA overflow), the error is DistilBERT-specific.

**Comment by EdwardZH** [comment]
It seems that the error caused by matrix multiplication. You can check the dimensions of  inputs_hidden https://github.com/thunlp/VERNet/blob/bd126d1e0596afe7de61d1b72797d72222e221b4/model/models.py#L85.

**Comment by The-One-Who-Speaks-and-Depicts** [comment]
Thank you very much for the response! The unfortunate thing, however, is that the model did not get to this point in code. I am sorry that I did not provided the context. The error happens here:
```
File ".../VERNet/model/train.py", line 91, in train_model
    score = model(inp_tensor, msk_tensor, seg_tensor)
```
after the following is called:
```
File ".../VERNet/model/train.py", line 165, in <module>
    train_model(model, ori_model, args, trainset_reader, validset_reader)
```

After some printing out, I met this line of code, that yielded the tensor of 243:
```
max_len = max_seq_length * 2 + 3
```

I changed this to `max_seq_length * 2 + 4`, and did the same with all the instances of `* 2 + 3`. After this, I changed max_len argument to 4. It seems to work both for bert, and distilbert. For bert, it works with the initial 120 max_len as well.

I presume there are some tradeoffs to it, but currently this at least seems to work for me. But I am not exactly a specialist, so I humbly wait for your opinion, maybe, there is a better solution?
